lib/codegen/fromcto/mermaid/mermaidvisitor.js

```diff
@@ -78,6 +79,32 @@ class MermaidVisitor extends DiagramVisitor {
         super.writeDeclarationSupertype(classDeclaration, parameters);
     }
 
+    /**
+     * Visitor design pattern
+     * @param {MapDeclaration} mapDeclaration - the object being visited
+     * @param {Object} parameters  - the parameter
+     * @param {string} type  - the type of the declaration
+     * @protected
+     */
+    visitMapDeclaration(mapDeclaration, parameters) {
+        let fqn = this.escapeString(mapDeclaration.getFullyQualifiedName());
+        let keyType   = mapDeclaration.getKey().getType();
+        let valueType = mapDeclaration.getValue().getType();
+
+        if (!ModelUtil.isPrimitiveType(keyType)) {
+            keyType = mapDeclaration.getModelFile().getFullyQualifiedTypeName(keyType);
+        }
+
+        if (!ModelUtil.isPrimitiveType(valueType)) {
+            valueType = mapDeclaration.getModelFile().getFullyQualifiedTypeName(valueType);
+        }
+
+        const type = `"1" ${DiagramVisitor.AGGREGATION} "*"`;
```

Comment:
maybe I'm misunderstanding, but I thought this would be a 1 to 1 as a map only has one of each key and value?

Reply:
@DianaLease the Many cardinality stems from a Map containing zero or more Entries of a Key and Value, so in this case cardinality is '*'. Though come to think of it, it may be more accurate to represent as "0..\*".

Reply:
Thanks, I will bring it up on the WG call tomorrow and see if there are any other opinions. But I think this is fine to merge as is and we can update if needed!

Reply:
Sounds good

Reply:
suggestion from the WG call - use a label to distinguish this as a map. Target of relationship is the value type for complex types.

Reply:
and make it 0..*